qiskit/scheduler/lowering.py

```diff
@@ -0,0 +1,95 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
```

Comment:
I think this should include the creation year of Qiskit (at least when I last asked Paul) but I might be wrong -- feel free to ignore 
```suggestion
# (C) Copyright IBM 2017, 2020.
```

Reply:
I see it done all sorts of ways in Qiskit